hystrix/hystrix_client.go

```diff
@@ -67,7 +67,7 @@ func NewClient(opts ...Option) *Client {
 	}
 
 	hystrix.ConfigureCommand(client.hystrixCommandName, hystrix.CommandConfig{
-		Timeout:                int(client.hystrixTimeout),
+		Timeout:                int(client.hystrixTimeout / 1000000),
```

Comment:
Is there a way to add a test and catch this ?

Reply:
@rShetty yea, can add basic test for this.

Reply:
The test would mostly be that 10*second would give a value of 10000 millisecond. Don't think there is any other way to test this.

Thoughts?

Reply:
also might be good to add a size cap; since int conversion of huge number might throw overflow exception.

Reply:
Yes, size cap is something which we can think about later. But a basic test and this PR looks good.

Reply:
@rShetty Have added tests